### PR TITLE
Frontend/feature/sudoku board #22

### PR DESCRIPTION
**Pull request by SzymonGlab**

### CODE REVIEW

**Comment by wadamek65** [comment]
It seems that you are creating a pull request to branch `master` but your branch is based on `frontend/feature/login-register-view` and this makes a lot of the changes between the two PRs duplicated. Consider changing the target branch to `frontend/feature/login-register-view` to make reviewing easier, and change it to master right before merge or after the other PR gets merged.

File: frontend/src/components/Board/Board.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+  }
+}
+
+Board.propTypes = {
+  fields: PropTypes.string
+};
+
+Board.defaultProps = {
+  fields:
+    "#################################################################################"
```
Comment (wadamek65) [inline frontend/src/components/Board/Board.js]: Are you sure you want to base your board on a string like this ? This is very fragile and can easily break if any change is made. Perhaps a simple two-dimensional array would work better?

File: frontend/src/components/Dragable/DragableField/DragableField.scss [hunk 2]
```diff
@@ -0,0 +1,32 @@
+
+@import "../../../Variables";
+
+.dragableField{
```
Comment (wadamek65) [inline frontend/src/components/Dragable/DragableField/DragableField.scss]: typo, should be: `draggable`, same everywhere else

File: frontend/src/components/Dragable/DragPanel/DragPanel.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    <table className="dragPanel">
+      <tbody>{rows}</tbody>
+    </table>
+  );
+};
+
+export default function DragPanel() {
+  return (
+    <div className="background">
+      <div> {generatePanel()} </div>
```
Comment (wadamek65) [inline frontend/src/components/Dragable/DragPanel/DragPanel.js]: Unnecessary div. If you need any properties from div you should apply it to the `dragPanel` css class.

File: frontend/src/models/BoardModel.js [hunk 4]
```diff
@@ -0,0 +1,34 @@
+import Field from "./FieldModel";
+
+export default class BoardModel {
+  constructor(input) {
+    let currentRow;
+    this.rows = [];
+
+    for (let idx = 0; idx < input.length; idx++) {
```
Comment (wadamek65) [inline frontend/src/models/BoardModel.js]: `const idx`

**Review by wadamek65 — APPROVED** [review]

File: frontend/src/models/BoardModel.js (hunk 4 above)
Comment (SzymonGlab) [reply frontend/src/models/BoardModel.js]: When I change 'let' to 'const' I get eslint  warning.

Line 8:45:  'idx' is constant  no-const-assign

Shall I remove this rule from linter?

Comment (wadamek65) [reply frontend/src/models/BoardModel.js]: I double checked it and it's fine. It should stay as it is, my bad.

**Review by wadamek65 — APPROVED** [review]

**Review by karolszymonczyk — APPROVED** [review]

**Pull request merged**